commands/validators.js

```diff
@@ -0,0 +1,118 @@
+const exitOnError = require('../utils/exit-on-error');
+const connect = require('../utils/connect');
+const inspectResponse = require('../utils/inspect-response');
+const bs58 = require('bs58');
+const { utils } = require('near-api-js');
+const BN = require('bn.js');
+const AsciiTable = require('ascii-table');
+
+function findSeatPrice(validators, numSeats) {
```

Comment:
this has way too many logic without explanation of what it is trying to do and why

also sounds like stuff like this should live in near-api-js, near-shell only taking care of displaying it as a table, etc

Reply:
see https://github.com/nearprotocol/near-shell/pull/328#issuecomment-619889936

Reply:
@ilblackdragon add some comment + TODO with link to GH issue then? Easy to lose context otherwise.